rpc, test: Fix JSON parsing errors in unloadwallet and getdescriptoractivity RPCs (#32845)
bitcoin/bitcoin: doc/release-notes-32845.md

Updated RPCs
------------

> - `unloadwallet` - Raise RPC_INVALID_PARAMETER describing that either the RPC endpoint
> or the wallet name parameter must be provided. Previously the RPC failed with a JSON
> parsing error.

Reviewer: ```suggestion
- `unloadwallet` - Return RPC_INVALID_PARAMETER when both the RPC wallet endpoint
and wallet_name parameters are unspecified. Previously the RPC failed with a JSON
parsing error.
```

Author: Done!
